src/v2/Apps/Conversation/Components/PurchaseOnInquiryButton.tsx

```diff
@@ -4,12 +4,15 @@ import { PurchaseOnInquiryButton_conversation } from "v2/__generated__/PurchaseO
 import { ConfirmArtworkButtonFragmentContainer } from "./ConfirmArtworkButton"
 import { useTracking } from "react-tracking"
 import { TappedBuyNow, ActionType, OwnerType } from "@artsy/cohesion"
+import { Button } from "@artsy/palette"
 
 export interface PurchaseOnInquiryButtonProps {
+  openInquiryModal: () => void
   conversation: PurchaseOnInquiryButton_conversation
 }
 
 export const PurchaseOnInquiryButton: React.FC<PurchaseOnInquiryButtonProps> = ({
+  openInquiryModal,
```

Comment:
Agreed. It's weird that it's called inquiry modal when this is to confirm an edition set:
```
openInquiryModal={() => setShowConfirmArtworkModal(true)}
```

It also seems we are prop-drilling this several levels down from the Conversation component.
It would be nice to use the same approach as the [useInquiry](https://github.com/artsy/force/blob/main/src/v2/Components/Inquiry/useInquiry.tsx) modal that opens whenever you need it without passing around any prop.